kubernetes/cray-sysmgmt-health/Chart.yaml

```diff
@@ -23,7 +23,7 @@
 #
 apiVersion: v2
 name: cray-sysmgmt-health
-version: 1.2.10
+version: 1.2.11
```

Comment:
Double check the version before merging, there are other PRs that also update the version